tests/coordinate_system_test.py

```diff
@@ -0,0 +1,87 @@
+#  Copyright (c) 2023 Mira Geoscience Ltd.
+#
+#  This file is part of geoh5py.
+#
+#  geoh5py is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  geoh5py is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with geoh5py.  If not, see <https://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+import numpy as np
+from PIL import Image
+
+from geoh5py.objects import GeoImage
+from geoh5py.workspace import Workspace
+
+# test tag
+tag = {
+    256: (128,),
+    257: (128,),
+    258: (8, 8, 8),
+    259: (1,),
+    33922: (0.0, 0.0, 0.0, 522796.33210329525, 7244067.563364625, 0.0),
+    42113: ("255",),
+    262: (2,),
+    33550: (0.9990415797117552, 0.999041579711816, 0.0),
+    339: (1, 1, 1),
+    277: (3,),
+    278: (5,),
+    284: (1,),
+    34737: ("WGS 84 / UTM zone 34N|WGS 84|",),
+}
+
+DEFAULT_CRS = {"Code": "Unknown", "Name": "Unknown"}
+
+
+def test_georeference_image(tmp_path):
```

Comment:
I don't think we need to be specific about image here. Any entity would do.

Reply:
Just do this test thinking of implementing it with the tag image but it's definitely for a next step. 
We could keep it there hopping we'll find a solution.

Reply:
Sounds good. just change the name of the test then, so it's not duplicating in our test suite